app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<h1> feed goes here </h1>
+
+<h1>STALK UR TEXT</h1>
+
+<h2> Welcome to the homepage You see when you are logged in! </h2>
+
+<h2>Welcome <%= @stalker.provider %> user!</h2>
```

Comment:
Hey, when you have a minute, can we chat about this? I'm confused about why this is in welcome#index rather than posts#index.